contrib/ci/fedora.sh

```diff
@@ -62,7 +69,7 @@ if [ -n "$CI" ]; then
 fi
 
 #build RPM packages
-rpmbuild -ba build/fwupd.spec
+rpmbuild -ba --define "qubes_packages $QUBES_MACRO" build/fwupd.spec
```

Comment:
I would rather see a `qubes_packages` set to zero by default in the spec file instead of "requiring" it set on command line.  Otherwise this spec file can't be used in Fedora anymore by "default".  If you can accomplish this it could become:

```
if [[ $1 == "qubes"* ]]; then
    QUBES="--define \"qubes_packages 1\""
endif
rpmbuild -ba $QUBES build/fwupd.spec
```

Reply:
Implemented. The change should look a little different:
```
if [ "$QUBES" = "true" ]; then
    QUBES_MACRO=(--define "qubes_packages 1")
fi
...
rpmbuild -ba "${QUBES_MACRO[@]}" build/fwupd.spec
```